recipe/meta.yaml

```diff
@@ -43,6 +63,11 @@ outputs:
         - typing-extensions
         - xarray >=0.17.0
         - xgboost >=1.6.0
+        # some tests fail if this pinning is not explicit
+        # because the version 2.8.3+snowflake will be picked
+        # up and apparently it's causing those tests fail
+        # - python-dateutil 2.8.2
```

Comment:
I'm still not a fan of this. Have we asked Snowflake's opinion? They are the one who provided us the patch to python-dateutil. If their patch breaks something, then we need to tell them.